### DISCUSSION TITLE
wine wayland

### DISCUSSION THREAD

**Discussion by huakim** [makedeb/prebuilt-mpr#174] [Issues]
### Package name

wine-wl-git

### List any other notes that might aid in the inclusion of this package (Is the upstream/MPR package popular? Is the upstream well-known or from a reputable source?).

This is a very popular package among GNOME users. A compatibility layer for running Windows programs on wayland.

**Comment by hwittenborn** [comment]
Is this on the MPR? It'll probably need to be packaged on their first if not.

This repository is mostly for asking existing MPR packages to be added to the Prebuilt-MPR. I'll go ahead and leave this open until someone responds though.

A request page on the MPR has also been asked for, I'll see if I can implement that sometime soon.

**Comment by hwittenborn** [comment]
I'm going to go ahead and close this for the time being @huakim. If you'd be willing to maintain the MPR package I'd be more than glad to help you in that process if needed. It's not something I see myself personally maintaining though, and I don't think it's going to get much tracking by asking in this repository (this isn't really the place for package requests, moreso only for adding existing MPR packages into the Prebuilt-MPR).
